demo/site/src/webpack/gql-hash-loader.ts

```diff
@@ -29,22 +29,25 @@ const gqlHashLoader = function (this: LoaderContext<unknown>, source: string) {
     const replacements: { start: number; end: number; replacement: string }[] = [];
     let match;
     while ((match = gqlTagRegex.exec(source)) !== null) {
-        const query = match[1];
+        const query = (match as RegExpExecArray)[1];
```

Comment:
`match` is inferred as `any`(probably due to the while-loop), so we need a type cast here.